### PR TITLE
The next web

### PR DESCRIPTION
**Pull request by Srjnnnn**
The page is responsive via grid and flexbox.
The page responds to the certain "max-width" properties via media queries.
The validator warned us about lack of headings on the sections but there is no heading on these sections. We tried to use semantics as much as we can.

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @eypsrcnuygr & @Haywhizzz  :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
## Main Page
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html)|
|:---:|:---:|
|tnw|tnw-clone|
- Remove the white border on the page's logo.
- Change the latest news header colour to its appropriate colour  

## Latest Funding Section
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html)|
|:---:|:---:|
|latest-funding|latest-funding-clone|
- Improve the styling on this section to match its source.

## Apps, Gears, Tech Section 
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html)|
|:---:|:---:|
|apps-gears|apps-gears-clone|
- Improve on the styling on this section as well.

## Latest Deals Section
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html)|
|:---:|:---:|
|latest-deals|latest-deals-clone|

- This section also needs some improvements as well

## Footer
|[Source Page](https://thenextweb.com/)|[Clone Page](https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html)|
|:---:|:---:|
|footer|footer-clone|
- Some more work needs to be carried out the footer as well.
##
## Other changes required
- The page is not really responsive and the changes are not being reflective as expected. In medium and small screens, the page's navbar needs to have a hamburger icon, this among other changes needs to be worked on. Kindly take note of all the changes needed for this project to be complete. Thank you!
## OPTIONAL
- Validate your markup using this [tool](https://validator.w3.org/nu/#textarea).
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by Srjnnnn** [comment]
The white border on the logo is removed,
Appropriate colors applied;
Border-tops applied,
Hamburger menu applied.
Footer two-color applied.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @eypsrcnuygr @Haywhizzz** 
Good job with your new submission for this project :muscle:  
Some improvements are required. Check out the list of changes your page needs:
- [x] This project requires a responsive behavior, it has to look like the source page not only in the large screen size both also in medium and small.
- [x] The small screen size version of your page looks like this:



### Latest news section:
- [x] For medium screen size it aligns 3 articles per row
- [x] For small screen size it aligns 1 article per row and each article has a cover image on the left side and title on the right side of the row.

### Latest funding section:
- [x] For medium screen size it aligns 3 items per row
- [x] For small screen size it aligns 1 item per row

### Categories (Apps, Gear, Tech, etc) section
- [x] For medium screen size it aligns 2 category boxes per row
- [x] For small screen size it aligns 1 category box per row

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Srjnnnn** [comment]
The latest news section,  latest funding, categories, and the last section are responsive and take the full width of the page for small screens.
Used flex and grid.
Used media queries for 765px and 1112 px.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status requires changes :recycle: 
## Hi guys, great job with the desktop view it looks good :clap:  but in other views, it's ruined :red_circle:, I see also that you have skipped most of the [required changes](https://github.com/Haywhizzz/TheNextWeb-Clone/pull/1#pullrequestreview-439887150) by the previous TSE please return back and fix them 

here is how your clone looks on mobile view:
https://raw.githack.com/Haywhizzz/TheNextWeb-Clone/TheNextWeb/index.html

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you are stuck or having any kind of questions don't hesitate to contact me on slack_

**Comment by Srjnnnn** [comment]
The page is responsive, for small, medium, and large screens.
There was a width issue, it was because of the rem unit and flex property. It is solved.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: required changes :recycle: 
## Hello guys it's me again I see that you have a lot to do in your project, first of all, fix your media queries cause your page breaks on some points where it shouldn't, also you need to work on the style see the comments below:

- [x] Fix the header on mobile view:
### clone:
image

### original:
image

- [x] Fix the header on tablet view also:
### clone:
image

### original:
image
 
- [x] on tablet view and smaller screens you need to remove the TNW logo from the main article and display it on the header:
### clone:
image
### original:
image

- [x] Latest article should show 4 articles on a row for desktop views:
### clone:
image

### original:
image

- [ ] on tablet view we should have three articles in a row but you have one messy article:
### clone:
image

### original:
image

actually the whole page is messy you need to fix your media queries guys:
image

### Please make the required changes before you submit another code review.
## Happy Coding :tophat: 
_If you are stuck or have any questions don't hesitate to contact me on slack_

**Comment by Srjnnnn** [comment]
There are two stages of media queries. One for max-width 806px and then 1025px . We think this cover all the necessary forms of the device widths'. Hope it's will pass.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @eypsrcnuygr @Haywhizzz** 
Nice job with the improvements since my last review :clap: :clap: :clap:  
There are still things to improve in different parts of the page before the project can be approved:
- [x] Make sure you are using the same responsive breakpoints of the original
- [x] You logo in the main news should disappear in medium screen size, your page:



Original page:



- [x] latest funding section should use 4 columns in large screen size, your page:



Original page:



- [x] latest funding section should use 3 columns in the medium screen size, your page:



The original page:




- [x] Your top three articles should be aligned 1 per row in small screen size (remember, use the same responsive breakpoints of the original). Your page:



The original:




Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Srjnnnn** [comment]
At the Next Web Page, the site responds at 1024 and 770px so we did the same. Hope to pass it.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: requires changes :recycle: 
## Hello, guys, it's me again you are almost there need to make just some more changes in order to make your project approved

- Fix the main section on tablet view:
### clone:
image


### original:
image

- [x] Fix the latest news section in tablet view it should look the same as the original:
### clone:
image

### original:
image

- [x] Fix the last funding section in tablet view
### clone:
image

### original:
image

- [x] Fix the categories section in tablet view:
### clone:
image

### original:
image

- [x] Fix the latest deals section on tablet view:
### clone:
image
### original:
image

### Please make the required changes before you request another code review.
# Happy Coding :tophat:

**Comment by Srjnnnn** [comment]
Before we choose 770px as the breakpoint but then it's understood that it should be 768px. So it's changed into 768px. Now for tablet view, it is displayed as the page.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Pull Request Under Review by Oscar Mier (TSE) 👨‍💻 🐦

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Nice work so far. 👍 
## However, there are some changes required 👀 🧰 🔨 🔧 
- On the Medium view of the 3 Top Articles section.
1. The TNW logo appears stretched. It's supposed to retain it's normal aspect ratio.
2. The navbar text wraps around. It's supposed to remain on a single line while shrinking the page.
3. The background gradient looks different from the original page.
image

- The Latest Funding and Categories sections should have the same width on Large display:
image

- They should also have the same width on Medium display:
image

- The footer text should be vertically centered and aligned with the TNW logo:
image

### [OPTIONAL]

- Use fonts that are more similar to the fonts from the original page. For this, you may use [Google Fonts](https://fonts.google.com/).

### Submit these changes and request a new code review! 😄 👍 
### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Comment by Srjnnnn** [comment]
For the background; to make it look like the original page we need to change the HTML structure which we avoid to do.
The latest funding and categories have the same issue as well. When we do this page, we tried different styles just to try ourselves with different properties. Now to align everything we give different margins and padding. That causes some differences on the look while you shrink, but for every type of device it responds well and it is aligned(Tablet view mobile view and big desktop view).

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Status: Changes required
Hello TEAM,** You have done a good job to create a similar skeleton of the original project and it is responsive too. But we have to work on very basic stylings like margin, padding, font-sizes, etc. I know it is a little bit demotivating to make a number of review requests and make changes. Let's focus on the basics and make sure that the project layout looks similar to the original one. For example, the font-size of the navbar is too small as compared to the original one, categories, and latest funding sections are not aligned properly. Please focus on one section at a time and make changes. 

**After making sure that it looks similar to the original one, submit the project for another review.
If you need any kind of help, I am happy to help you. Just DM me on slack Suman Shrestha.
Happy coding !!!**

**Comment by Srjnnnn** [comment]
Throughout the page, the CSS is arranged using grid. 
Everything is aligned properly. 
Font sizes arranged. Font size for the navbar can't be more otherwise it doesn't fit into one line.
Media queries in 1024 and 768.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello team 👋. Your project looks impressive and you have done a great job implementing the changes. 

### However, we still need to work on some more changes before we can merge.

- Let's fix the top section:
  - The logo is different than the original.
  #### Clone

  image

  #### Original
  
  image

  - The `tnw-container` class looks different.

  #### Clone

  image

  #### Original

  image

  - Let's fix this part:
  
     image

- Let's reduce the space here:

  image

- The social icons in the footer must be links. Please do that.

Apply these changes and request another review.

Happy coding 👨‍💻

**Comment by Srjnnnn** [comment]
The background is the same as the page now
Unnecessary margins deleted
Footer links added

**Comment by Srjnnnn** [comment]
The background is the same as the page now
Unnecessary margins deleted
Footer links added

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hi, it's me again 😀. Thanks for making the changes. You are almost there.

- Let's decrease the height of `first-section` class.

- Let's make the `latest funding` section more like the original.

#### Clone
Annotation 2020-07-03 021503

#### Original
Annotation 2020-07-03 021517

- Let's give this section some breathing room. Please add some padding to the list items.
image

- Let's fix this 👇
Annotation 2020-07-03 021819

#### Here is the original
Annotation 2020-07-03 021847

- Same in `latest deals` 👇
Annotation 2020-07-03 022017

#### Here is the original
Annotation 2020-07-03 022038

Apply these changes and request another review.

Happy coding 👨‍💻

**Comment by Srjnnnn** [comment]
Latest funding arranged
Apps gears arranged
The upper part was given more space
There was a yellow sticky nav component before, the site removed it so we did as well

**Comment by Haywhizzz** [comment]
Latest funding arranged
Apps gears arranged
The upper part was given more space
There was a yellow sticky nav component before, the site removed it so we did as well
The original page structure changes every 1-2 hours and some of the issues pointed out here are due to that.

File: assets/styles.css [hunk 1]
```diff
@@ -0,0 +1,814 @@
...
+  height: 50px;
+  width: 50px;
+  margin-right: 15px;
+}
+
+.footer-end img {
+  width: 50px;
+  height: 50px;
+  margin-right: 15px;
+}
```
Comment (andresporras3423) [inline assets/styles.css]: Group selectors with the same style definition, or create a new class.

File: assets/styles.css [hunk 2]
```diff
@@ -0,0 +1,814 @@
...
+
+.p-tag {
+  border-top: 1px solid pink;
+  font-size: 0.7em;
+}
+
+.flex {
+  display: flex;
+}
```
Comment (andresporras3423) [inline assets/styles.css]: Same style definition of the selector on line 289. Group selectors, or create a new class.

File: assets/styles.css [hunk 3]
```diff
@@ -0,0 +1,814 @@
...
+
+.last-grid-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.non-flex-section3 {
+  display: block;
+}
```
Comment (andresporras3423) [inline assets/styles.css]: Same style definition of the selector on line 462. Group selectors, or create a new class.

File: assets/styles.css [hunk 4]
```diff
@@ -0,0 +1,814 @@
...
+    font-size: 1.1em;
+  }
+
+  .flex-div * {
+    font-size: 1em;
+  }
+
+  .search-button {
+    display: block;
+  }
```
Comment (andresporras3423) [inline assets/styles.css]: Same style definition of the selector on line 642. Group selectors, or create a new class.

File: assets/styles.css [hunk 5]
```diff
@@ -0,0 +1,814 @@
...
+    font-size: 0.5em;
+  }
+
+  .second-headings {
+    grid-column-end: 4;
+  }
+
+  #none {
+    display: none;
+  }
```
Comment (andresporras3423) [inline assets/styles.css]: Same style definition of selectors on lines 780 and 790. Group selectors or create a new class.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Haywhizzz @eypsrcnuygr** 
Good job with your new submission for this project :muscle:  
the appearance of your page almost math the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.
- [x]  For categories section, on medium version size, it must align 2 category boxes per row. Your page:



The original:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Srjnnnn** [comment]
For the medium view, the part is arranged as two columns.
Repeated styles are grouped together.

File: assets/styles.css [hunk 6]
```diff
@@ -0,0 +1,778 @@
...
+.fa-clock,
+.fa-share-alt,
+.fa-rss,
+.fa-envelope {
+  color: white;
+}
+
+.fab {
+  color: white;
+}
```
Comment (andresporras3423) [inline assets/styles.css]: Group selectors with the same style definition, or create a new class.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @eypsrcnuygr @Haywhizzz** 
Good job with the improvements for this project :muscle:  
Your project passes most of the requirements :clap: :clap: :clap:  
Just a last detail to solve before you can be approved:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.

### OPTIONAL

- [x] It's a good practice to have a folder for the images and a different one for the styles

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by Srjnnnn** [comment]
Style folder added and repeated style fab is grouped.
Looked for grouped styles, none found.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job team :+1:
- You can now merge the changes.
-  I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**